Tests/Unit/DependencyInjection/CmfSeoExtensionTest.php

```diff
@@ -121,11 +121,13 @@ public function testAlternateLocaleWithPhpcr()
             'Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\AlternateLocaleProvider'
         );
 
-        $this->assertContainerBuilderHasServiceDefinitionWithMethodCall(
-            'cmf_seo.event_listener.seo_content',
-            'setAlternateLocaleProvider',
-            array($this->container->getDefinition('cmf_seo.alternate_locale.provider_phpcr'))
-        );
+        if ($this->container->has('cmf_seo.event_listener.seo_content')) {
```

Comment:
normally the alternate locale provider is injected into the SeoPresentation, when activated and with phpcr as implementation, this is done inside of the `CmfSeoExtension`. I would propose to add your changes there.